src/client/components/FormFestivals.js

```diff
@@ -50,6 +50,12 @@ const FormFestivals = () => {
         name="documents"
         validate={schema.documents}
       />
+
+      <InputUploadField
+        label={translate('FormFestivals.fieldArtworks')}
+        name="artworks"
+        validate={schema.artworks}
+      />
```

Comment:
`InputUploadField` for artworks? Looks like this is a leftover from something else.

Reply:
It's more just a placeholder until I get search working

Reply:
(which I was planning for the next pr)